MediaManager/Media/MediaLocation.cs

```diff
@@ -2,7 +2,7 @@
 {
     public enum MediaLocation
     {
-        Unknown,
+        Unknown = 0,
```

Comment:
Do all need to have a value?

Reply:
I did that to be sure that `Unknown` is the default value.